release.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -euo pipefail
+
+NEW_VERSION="$(date +%Y.%m.%d)"
+OLD_VERSION="$(git tag --list | tail -n 1)"
```

Comment:
Agreed. It's possible that the fetch-and-merge could introduce new tags onto the master branch, so we should do the fetch-and-merge before setting `OLD_VERSION`.